content/test/unit/repository/DeploymentsRepository.spec.ts

```diff
@@ -0,0 +1,456 @@
+        })
+
+        it('should call the database with the expected sorting', async () => {
+          await repository.getHistoricalDeployments(
+            0,
+            10,
+            { from: 1, to: 11 },
+            {
+              field: SortingField.ENTITY_TIMESTAMP,
+              order: SortingOrder.ASCENDING
+            },
+            lastId
+          )
+
+          const args = capture(db.map).last()
+
+          expect(args[0]).toContain(`ORDER BY dep1.entity_timestamp ASC`)
+          expect(args[0]).toContain(`((LOWER(dep1.entity_id) > LOWER($(lastId))`)
+          expect(args[0]).toContain(
+            `dep1.entity_timestamp = to_timestamp($(fromEntityTimestamp) / 1000.0)) ` +
+              `OR (dep1.entity_timestamp > to_timestamp($(fromEntityTimestamp) / 1000.0))) ` +
+              `AND dep1.entity_timestamp <= to_timestamp($(toEntityTimestamp) / 1000.0)`
+          )
+
+          expect(args[1]).toEqual(jasmine.objectContaining({ fromEntityTimestamp: 1, toEntityTimestamp: 11 }))
+        })
+      })
+
+      describe("when it doesn't receive a field or order to sort by", () => {
+        beforeEach(() => {
+          when(db.map(anything(), anything(), anything())).thenReturn(Promise.resolve([]))
+        })
+
+        it('should call the database with the default sorting', async () => {
+          await repository.getHistoricalDeployments(0, 10, { from: 1, to: 11 }, undefined, lastId)
+
+          const args = capture(db.map).last()
+
+          expect(args[0]).toContain(`ORDER BY dep1.local_timestamp DESC`)
+          expect(args[0]).toContain(`((LOWER(dep1.entity_id) < LOWER($(lastId))`)
+          expect(args[0]).toContain(
+            `dep1.local_timestamp >= to_timestamp($(fromLocalTimestamp) / 1000.0) AND ` +
+              `((LOWER(dep1.entity_id) < LOWER($(lastId)) AND dep1.local_timestamp = to_timestamp($(toLocalTimestamp) / 1000.0)) ` +
+              `OR (dep1.local_timestamp < to_timestamp($(toLocalTimestamp) / 1000.0)))`
+          )
+
+          expect(args[1]).toEqual(jasmine.objectContaining({ fromLocalTimestamp: 1, toLocalTimestamp: 11 }))
+        })
+      })
+    })
+
+    describe('when there is a deployed by filter', () => {
+      beforeEach(() => {
+        db = mock(MockedDataBase)
+        repository = new DeploymentsRepository(instance(db) as any)
+
+        when(db.map(anything(), anything(), anything())).thenReturn(Promise.resolve([]))
+      })
+
+      it('should add the expected where clause to the query with the addresses on lowercase', async () => {
+        const deployedBy = ['jOn', 'aGus']
+        await repository.getHistoricalDeployments(0, 10, { deployedBy })
+
+        const args = capture(db.map).last()
+
+        expect(args[0]).toContain(`LOWER(dep1.deployer_address) IN ($(deployedBy:list))`)
+        expect(args[1]).toEqual(jasmine.objectContaining({ deployedBy: deployedBy.map((x) => x.toLowerCase()) }))
```

Comment:
nit: as this is the expected param, shouldn't it be hardcoded?

Reply:
Updated to have it hardcoded